src/components/LayoutManager/Controller/LayoutWorker.js

```
@@ -1,11 +1,32 @@
 
-    if (e.data == "hello") {
-        self.postMessage("hello from worker!")
+    try {
+        const args = e.data.args;
+        switch (e.data.code) {
+            case LAYOUT_WORKER_PROTOCOL.INITIALIZE:
+                /** @type {LayoutEditor} */
+                editor = new LayoutEditor(e.data.args.ldf);
+                break;
+            case LAYOUT_WORKER_PROTOCOL.RENDER_NODE:
+                const d = e.data.args;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)